tests/unit/nupic/research/spatial_pooler_compatability_test.py

```diff
@@ -389,6 +389,27 @@ def testSerializationRun(self):
     self.runSerialize("cpp", params)
 
 
+  def testInhibitColumnsGlobal(self):
+    params = {
+      "inputDimensions": [512],
+      "columnDimensions": [512],
+      "globalInhibition": True,
+      "numActiveColumnsPerInhArea": 40,
+      "seed": 19
```

Comment:
nit: usually good to put trailing comma even for last element. Keeps diff clean when modifying sequences.